tools/RAiDER/processWM.py

```diff
@@ -6,18 +6,14 @@
 #  RESERVED. United States Government Sponsorship acknowledged.
 #
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-import contextlib
 import os
-import sys
-
 import numpy as np
-import matplotlib.pyplot as plt
-import xarray as xr
-
-from datetime import datetime, date
-
-from RAiDER.logger import *
+import rasterio
```

Comment:
Probably we should keep the original syntax (spacing) here to be consistent with the other modules, and update them all together if desired.

Reply:
Is there a static analysis tool I should be using? I can just turn off flake8 too.

Reply:
There is an autopep8 cmd we've been running (see the template PR), otherwise not.